From 5f2cfd8c567beba4327fb6402c7b347ba14c4c34 Mon Sep 17 00:00:00 2001
From: Adrian Stratulat <adrian.stratulat91@gmail.com>
Date: Sat, 29 Feb 2020 13:15:14 +0200
Subject: [PATCH] Update wsjtx interface to keep it synced with hamlib

---
 HamlibTransceiver.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/HamlibTransceiver.hpp b/HamlibTransceiver.hpp
index bcc040d2..a1c10848 100644
--- a/HamlibTransceiver.hpp
+++ b/HamlibTransceiver.hpp
@@ -14,7 +14,7 @@ extern "C"
 {
   typedef struct rig RIG;
   struct rig_caps;
-  typedef int vfo_t;
+  typedef unsigned int vfo_t;
 }
 
 // hamlib transceiver and PTT mostly delegated directly to hamlib Rig class
-- 
2.17.1

